study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/RarexVersion2.java

```diff
@@ -0,0 +1,723 @@
+                        + ".hasOption(\"SELF\")\n            && user.studies[\"rarex\"].forms[\"HEALTH_AND_DEVELOPMENT\"]."
+                        + "questions[\"HD_GENETIC_TEST\"].answers.hasOption(\"YES\")");
+        updateExpressionTextByQuestionStableCode(healthAndDevelopmentId, "HD_GENETIC_TEST_REPORT",
+                "user.studies[\"rarex\"].forms[\"ADD_PARTICIPANT\"].questions[\"PARTICIPANT_RELATIONSHIP\"].answers."
+                        + "hasOption(\"SELF\")\n            && user.studies[\"rarex\"].forms[\"HEALTH_AND_DEVELOPMENT\"]."
+                        + "questions[\"HD_GENETIC_TEST\"].answers.hasOption(\"YES\")");
+        updateExpressionTextByQuestionStableCode(healthAndDevelopmentId, "HD_GENETIC_TEST_REPORT_PATIENT",
+                "!user.studies[\"rarex\"].forms[\"ADD_PARTICIPANT\"].questions[\"PARTICIPANT_RELATIONSHIP\"].answers."
+                        + "hasOption(\"SELF\")\n            && user.studies[\"rarex\"].forms[\"HEALTH_AND_DEVELOPMENT\"]."
+                        + "questions[\"HD_GENETIC_TEST\"].answers.hasOption(\"YES\")");
+        updateExpressionText(healthAndDevelopmentId,
+                "user.studies[\"rarex\"].forms[\"ADD_PARTICIPANT\"].questions[\"PARTICIPANT_RELATIONSHIP\"].answers."
+                        + "hasOption(\"SELF\")\n            && user.studies[\"rarex\"].forms[\"HEALTH_AND_DEVELOPMENT\"]."
+                        + "questions[\"HD_GENETIC_TEST\"].answers.hasAnyOption(\"YES\",\"UNSURE\")",
+                "user.studies[\"rarex\"].forms[\"ADD_PARTICIPANT\"].questions[\"PARTICIPANT_RELATIONSHIP\"].answers."
+                        + "hasOption(\"SELF\")\n            && user.studies[\"rarex\"].forms[\"HEALTH_AND_DEVELOPMENT\"]."
+                        + "questions[\"HD_GENETIC_TEST\"].answers.hasOption(\"YES\")");
+        updateExpressionText(healthAndDevelopmentId,
+                "user.studies[\"rarex\"].forms[\"ADD_PARTICIPANT\"].questions[\"PARTICIPANT_RELATIONSHIP\"].answers."
+                        + "hasAnyOption(\"PARENT\", \"GUARDIAN\")\n            && user.studies[\"rarex\"]"
+                        + ".forms[\"HEALTH_AND_DEVELOPMENT\"].questions[\"HD_GENETIC_TEST\"].answers.hasAnyOption(\"YES\",\"UNSURE\")",
+                "user.studies[\"rarex\"].forms[\"ADD_PARTICIPANT\"].questions[\"PARTICIPANT_RELATIONSHIP\"].answers."
+                        + "hasAnyOption(\"PARENT\", \"GUARDIAN\")\n            && user.studies[\"rarex\"]"
+                        + ".forms[\"HEALTH_AND_DEVELOPMENT\"].questions[\"HD_GENETIC_TEST\"].answers.hasOption(\"YES\")");
+        updateExpressionText(healthAndDevelopmentId,
+                "user.studies[\"rarex\"].forms[\"HEALTH_AND_DEVELOPMENT\"].questions[\"HD_GENETIC_TEST\"].answers."
+                        + "hasAnyOption(\"YES\",\"UNSURE\")",
+                "user.studies[\"rarex\"].forms[\"HEALTH_AND_DEVELOPMENT\"].questions[\"HD_GENETIC_TEST\"].answers."
+                        + "hasOption(\"YES\")");
+
```

Comment:
I recommend working through revisioning each activity one at a time. For each activity, create new version tag/revision ids, making the changes as part of that new revision.